shard.yml

```diff
@@ -1,13 +1,12 @@
 name: invidious
-version: 0.20.1
+version: 2.20240825.2-dev
 
 authors:
   - Omar Roth <omarroth@protonmail.com>
-  - Invidious team
+  - Invidious team <contact@invidious.io>
 
-targets:
-  invidious:
-    main: src/invidious.cr
```

Comment:
Why removing that, though?

Reply:
I just modeled the file after the Crystal compiler one which doesn't have the target section.

Plus Invidious isn't intended to be built via shards